Kaha-Mason/Kaha-Mason.md

```diff
@@ -6,20 +6,36 @@ qualifications:  "Bachelor of Information Technology"
 quote:  "Coding is like artwork and as developers we should always strive to be better artists."
 ---
 
-Kaha is a software developer based in SSW's Brisbane Office. Kaha has been involved with development in a range of Angular and .NET based projects and has experience developing in client-side, server-side and SQL database environments. Kaha is versed in Scrum and operates well in team-oriented environments, interacts well with clients and development teams and adapts well to development changes.  
+[[imgBadge]] | ![dotnetcore.png](../badges/Developer-dotnet-core.png)
 
-Some of these skills include:  
+[[imgBadge]] | ![dotnetcode.png](../badges/Developer-dotnet-code.png)
```

Comment:
Hi Kaha,
can you change all the badges from:
```
[[imgBadge]] | ![dotnetcode.png](../badges/Developer-dotnet-code.png)
```
To
```
[[imgBadge]]
| ![dotnetcode.png](../badges/Developer-dotnet-code.png)
```

Reply:
Done with latest commit